### ISSUE TITLE
Legendaries daggers progression not working. Parte 1: Las mandíbulas de la represión, el miedo y la venganza

### ISSUE THREAD

**Issue by watchman929** [gamefreedomgit/Maelstrom#6208]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (https://www.wowhead.com/cata)

**Description:**
The quest "[A hide message](https://www.wowhead.com/es/quest=29802/un-mensaje-escondido)" I already  got the decoded ring "https://www.wowhead.com/es/item=74256/anillo-descodificador-de-criptom%C3%A1ntico" and get the item needed to complete the quest, however, at the moment to complete it, is not working, it is bug and it shows the progression as in the picture submitted below. 

**How to reproduce:**

**How it should work:**

**Database links:**
![legendaries dagger bugged](https://github.com/user-attachments/assets/c3dafa8a-358f-42fd-8490-b43f0854f1d1)

**Comment by Sketish** [comment]
I was able to complete quest on ptr without any issues. Can you provide more details ?

**Comment by watchman929** [comment]
I only tried it once because I dont have enough gold to try to start over the quest again, And it seems like I will have to abandon the quest and pick it up again to try it, but I spent the only 10k gold trying to complete the quest once. by the way, i saw another person with the same issue as mine.

**Comment by Sketish** [comment]
Since your client is spanish what credit of quest are you missing ? https://www.wowhead.com/item=74748/charged-decoder-ring this one ?

**Comment by watchman929** [comment]
Yes, that is the one that i'm missing, the charged decoded ring. I paid 10k to charged it and it didnt complete the quest.

**Comment by Sketish** [comment]
You are getting this item from np after paying 10k. It wasnt added to your bag or you you got item and didn;t get credit ?

**Comment by watchman929** [comment]
Actually, I paid 10k for the ring and I get the item, but at the time to complete the quest to the NPC as I submitted on the picture above, it didnt work, like if I never had the item, it dessapeared.

**Comment by PanJanuszek** [comment]
remove and take quest again. non English clients have some issues with some quests

**Comment by watchman929** [comment]
Ok, I can try to do the quest again, but the 10k spent is gonna be Lost?

**Comment by watchman929** [comment]
Ok, I've spent 20k gold and the quest still bugged, I dont have more gold to waste doing the quest, so it means im not able to get the daggers, neither my gold back because of the bugged quest.... OMG!

**Comment by Sketish** [comment]
Please make ticket in game and GM will assist you with that quest.

**Comment by watchman929** [comment]
I have already created 3 tickets in Game, on the first one the GM just sent the link of this page and on the others two, they were deleted for no reason and I didnt received any solution.

**Comment by GMNeon** [comment]
The issue with this quest is that the Charging decoder ring transforms into the charged decoder ring after 1 hour but the quest does not understand that the ring has changed. once the cipher is solved the ring is then used, but the sipher is solved.
